components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadNotificationTest.kt

```diff
@@ -29,9 +23,9 @@ class DownloadNotificationTest {
     fun getProgress() {
         val downloadJobState = DownloadJobState(
                 job = null,
-                state = DownloadState(url = "mozilla.org/mozilla.txt", contentLength = 100L),
-                currentBytesCopied = 10,
-                status = ACTIVE,
+                state = DownloadState(url = "mozilla.org/mozilla.txt", contentLength = 100L,
+                        currentBytesCopied = 10,
+                        status = DownloadState.Status.DOWNLOADING),
```

Comment:
nit: We can use import `DownloadState.Status.DOWNLOADING` and just use `DOWNLOADING`